tests/installation/accept_license.pm

```diff
@@ -20,13 +20,15 @@
 use strict;
 use base "y2logsstep";
 use testapi;
-use version_utils qw(is_sle sle_version_at_least);
 
 sub run {
     my ($self) = @_;
 
     # workaround for bsc#1069124: Skip the license check in upgrade mode
-    return record_soft_failure('bsc#1069124: License is not shown in upgrade') if is_sle && sle_version_at_least('15') && get_var('UPGRADE');
+    # During upgrade to sle 15, license agreement is shown on aarch64 and s390x
+    # but not shown on ppc64le and x86_64 at the moment
+    return record_soft_failure('bsc#1069124: License is not shown in upgrade')
+      if get_var('UPGRADE') && (check_var('ARCH', 'ppc64le') || check_var('ARCH', 'x86_64'));
```

Comment:
I don't think removing the version check is a good idea even if it's only triggered for SLE15 now. The reason is that we should be safe to only call the record_soft_failure on versions for which the bug was reported. And even I think it would be cleaner if we just trigger this test module in all versions but only do the relevant part depending on the version.
What do you think?